docs/querying/api.md

```diff
@@ -381,3 +381,85 @@ $ curl http://localhost:9090/api/v1/alertmanagers
   }
 }
 ```
+
+
+## v2 TSDB Admin APIs
+With the new storage engine in 2.0, we also instroduced new APIs that expose database functionalities for the advanced user. These APIs are not enabled unless the `--web.enable-admin-api` is set. All the APIs below are experimental and might change in the future.
+
+We also expose a gRPC API whose definition can be found [here](https://github.com/prometheus/prometheus/blob/master/prompb/rpc.proto).
+
+### Snapshot
+Snapshot creates a snapshot of all current data into `snapshots/<datetime>-<rand>` under the TSDB's data directory and returns the directory as response.
+
+```
+POST /api/v2/admin/tsdb/snapshot
+```
+
+```json
+$ curl http://localhost:9090/api/v2/admin/tsdb/snapshot
+{
+  "name": "2017-11-30T15:31:59Z-2366f0a55106d6e1"
+}
+```
+
+The snapshot now exists at `<data-dir>/snapshots/2017-11-30T15:31:59Z-2366f0a55106d6e1`
+
+
+### Clean Tombstones
+CleanTombstones removes the deleted data from disk and cleans up the existing tombstones. This can be used in conjunction with delete series to implement dynamic retention or to just free up space when there are deleted series.
```

Comment:
I'm not sure I'd mention dynamic retention here, as this isn't a feature we support directly and I can it causing confusion.